coralproject/talk
# [CORL-901] Rebrand comment actions and body

I need to bug Kiwi/Sam about feasibility of some internationalization issues around the responsive action buttons in the latest Figma specs. After that's done this can be fully reviewed and merged. Putting as draft until I can sort that out, everything else is good though.

## What does this PR do?

Rebrands the comment bodies, actions, and user information to match the new branding.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Head to comment stream
- See new branding styles in the comment body text, commenter info, and action buttons.
